docs/docs/developers/page-editor/index.md

```diff
@@ -9,6 +9,20 @@ content and configuration to the _components_.
 
 ![Page Editor](images/page-editor-overview.png)
 
+## Component actions
+
```

Comment:
Those are the default actions provided by the `specific` module. The list is extendable. So instead of `Available actions` we should use smth like `List of default actions from </module>`

Reply:
I think that we should not mention any modules here. I set this to 'List of actions available by default'. I will add here info about extending the list once we document it.